src/borg/testsuite/archive.py

```diff
@@ -356,3 +357,9 @@ def test_get_item_uid_gid():
     # because item uid/gid seems valid, do not use the given uid/gid defaults
     assert uid == 9
     assert gid == 10
+
+
+def test_reject_non_sanitized_item():
+    for path in rejected_dotdot_paths:
+        with pytest.raises(ValueError, match="unexpected '..' element in path"):
+            Item(path=path, uid=0, gid=0, user="root", group="root")
```

Comment:
is it necessary to giver user and group here or is uid and gid enough?

asking because gid 0 on bsd and macOS is group "wheel".

Reply:
Yes, user and group is enough. Changed.